src/firefly/js/ui/VersionInfo.jsx

```diff
@@ -49,28 +52,47 @@ function VersionInfoFull() {
         + (ffTag ? `\n${FF_TAG}:         ${ffTag}` : '');
 
     return (
-        <div style={{display: 'flex', justifyContent: 'space-evenly'}}>
-            <div className='Version-grid'>
-                <Entry desc={VER} value={version}/>
-                <Entry desc={BUILT_ON} value={BuildTime}/>
-                <Entry desc={COMMIT} value={BuildCommit}/>
-                {major ? <Entry desc={FF_LIB} value={getFireflyLibraryVersionStr()}/> : ''}
-                {ffCommit && <Entry desc={FF_COM} value={ffCommit}/>}
-                {ffTag && <Entry desc={FF_TAG} value={ffTag}/>}
-                {
-                    !isVersionFormalRelease() &&
-                    <div className='Version-grid-warning'>
-                        {isVersionPreRelease() ?
-                            `Warning: Early preview of Firefly ${getDevCycle()}` :
-                            `Warning: Development build of Firefly on dev cycle ${getDevCycle()}`
-                        }
+        <div style={{display: 'flex', justifyContent: 'space-evenly', flexDirection:'column'}}>
+            <div style={{display: 'flex', justifyContent: 'flex-start'}}>
+                <div style={{paddingRight:10, display: 'flex', flexDirection:'column', alignItems:'center', marginTop:-20}}>
+                    <a href='https://github.com/Caltech-IPAC/firefly' target='github-window'><img alt='Firefly' src={FFTOOLS_ICO} style={{marginTop:10}}/></a>
+                    <a href='https://www.caltech.edu' target='caltech-window'><img style={{width:70}} alt='Caltech' src={CALTECH_LOGO}/></a>
+                    <a href='https://www.ipac.caltech.edu' target='ipac-window'><img alt='IPAC' src={IPAC_LOGO}/></a>
+                </div>
+                <div style={{display: 'flex', justifyContent: 'space-evenly', marginLeft:60, alignItems:'flex-start'}}>
+                    <div className='Version-grid'>
+                        <Entry desc={VER} value={version}/>
+                        <Entry desc={BUILT_ON} value={BuildTime}/>
+                        <Entry desc={COMMIT} value={BuildCommit}/>
+                        {major ? <Entry desc={FF_LIB} value={getFireflyLibraryVersionStr()}/> : ''}
+                        {ffCommit && <Entry desc={FF_COM} value={ffCommit}/>}
+                        {ffTag && <Entry desc={FF_TAG} value={ffTag}/>}
```

Comment:
I'm not quite sure how the copy-to-clipboard icon in this dialog works, but I just want to be sure that it only copies the version info and not the acknowledgement sentence.  (I don't have VPN access at the moment to test it live.)

Reply:
It only copies the version. Nothing has changed.

Reply:
yes it does only copy the version.

Reply:
Thanks!